internal/resend/src/client.tsx

```diff
@@ -58,26 +63,37 @@ export class Resend {
       }
       throw result.error;
     } catch (error) {
-      console.error("Error occurred sending subscription email ", JSON.stringify(error));
+      console.error(
+        "Error occurred sending subscription email ",
+        JSON.stringify(error)
+      );
     }
   }
 
   public async sendWelcomeEmail(req: { email: string }) {
+    const fiveMinutesFromNow = new Date(
```

Comment:
using a string.... what the hell was resend thinking?

Reply:
Me: this is an odd choice. 
Me: Let me read the doc again. 
Me: This is an odd choice.